xds/src/test/java/io/grpc/xds/XdsNamResolverTest.java

```diff
@@ -66,6 +69,23 @@ public void uncaughtException(Thread t, Throwable e) {
           .build();
 
   private final XdsNameResolverProvider provider = new XdsNameResolverProvider();
+  private static final Node bootstrapNode = Node.newBuilder().setBuildVersion("fakeVer").build();
+  private static final Bootstrapper bootstrapper = new Bootstrapper() {
```

Comment:
Remove `static`.

Reply:
Why?

Reply:
(This is somewhat controversial). Usually I (almost, may missed some) only use `static final` for something that has a strong reason to be constant (therefore, names capitalized). They are initialized before the enclosing class is loaded. For here, not a strong reason to do that.

Reply:
I don't have strong opinion about that.
- `assertBootstrapServiceConfig()` is static and is using `bootstrapNode`, so `bootstrapNode` will stay static.
- `bootstrapper` does not need to be a field. Moved to a local variable.

Reply:
Ok. Then treat it as a constant, use all-capitalized name, move it above non-static fields.

Reply:
Done.